src/ConfigDiscovery/AbstractDiscovery.php

```diff
@@ -0,0 +1,59 @@
+<?php
+
+declare(strict_types=1);
+
+namespace Mezzio\Tooling\ConfigDiscovery;
+
+use function file_get_contents;
+use function is_dir;
+use function is_file;
+use function preg_match;
+use function sprintf;
+
+/**
+ * @internal
+ */
+abstract class AbstractDiscovery implements DiscoveryInterface
```

Comment:
Tests for these symbols should also be imported 🤔

Reply:
Sorry, I totally missed that in my previous review :(

Reply:
We do actually test it by integration testing the whole component (e.g. that the `ConfigProvider` is injected to the appropriate file on the FS). 

Do you still want me to unit-test these files even if its covered by integration tests?
just give me a quick thumbs up if so, if not, thumbs down.

Reply:
Mostly just copying over the tests from the other components would suffice.

Reasoning is that as we apply CS fixes and static-analysis related changes/refactoring, if we have coverage, we're OK. Very risky otherwise.

Reply:
Done!